phpunit.xml.dist

```diff
@@ -1,15 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<phpunit backupGlobals="false"
-         backupStaticAttributes="false"
-         bootstrap="./vendor/autoload.php"
-         colors="true"
-         convertErrorsToExceptions="true"
-         convertNoticesToExceptions="true"
-         convertWarningsToExceptions="true"
-         processIsolation="false"
-         stopOnFailure="false"
-         syntaxCheck="false"
+<phpunit
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
```

Comment:
Related, why are we requiring

```json
{
    require: {
        "phpunit/phpunit": "^4.8.35 || ^5.4.3",
    }
}
```

instead of only `^5.4.3` or even higher?

After all, we require at least PHP 5.6, could surely use something more current for `phpunit/phpunit`!